source/Calamari/Kubernetes/Commands/KubernetesApplyRawYamlCommand.cs

```diff
@@ -93,10 +96,19 @@ public override int Execute(string[] commandLineArguments)
                 new ConfigurationTransformsConvention(new ConfigurationTransformsBehaviour(fileSystem, variables, configurationTransformer, transformFileLocator, log)),
                 new ConfigurationVariablesConvention(new ConfigurationVariablesBehaviour(fileSystem, variables, replacer, log)),
                 new StructuredConfigurationVariablesConvention(new StructuredConfigurationVariablesBehaviour(structuredConfigVariablesService)),
+            };
+
+            if (variables.Get(Deployment.SpecialVariables.Account.AccountType) == "AmazonWebServicesAccount")
```

Comment:
Is there somewhere where the "AmazonWebServicesAccount" constant can be defined and referenced here?

Reply:
I have just gone through and updated all magic strings for Account types to consts but it touches a few files so I might do it as a separate PR if that's ok with you. Good call out though 👍